src/ws/Makefile-ws.am

```diff
@@ -167,16 +167,19 @@ nodist_systemdunit_DATA += \
 firewalldir = $(prefix)/lib/firewalld/services
 firewall_DATA = src/ws/cockpit.xml
 
-issuedir = $(datadir)/$(PACKAGE)/issue/
-issue_DATA = \
-	src/ws/inactive.issue \
-	src/ws/active.issue \
-	$(NULL)
+motddir = $(datadir)/$(PACKAGE)/motd/
+dist_motd_DATA = src/ws/inactive.motd
+dist_motd_SCRIPTS = src/ws/activate-motd
 
 tempconfdir = $(prefix)/lib/tmpfiles.d
 nodist_tempconf_DATA = src/ws/cockpit-tempfiles.conf
 tempconf_in = src/ws/cockpit-tempfiles.conf.in
 
+# Automake: 'Variables using ... ‘sysconf’ ... are installed by install-exec.'
```

Comment:
What does this comment mean?

Reply:
it's an excerpt from the autoconf manual explaining why I'm installing a config file with install-exec-hook instead of install-data-hook or some other hook.